### PR TITLE
Bunch of clock stuff

### PR DESCRIPTION
**Pull request by Readystorm546**
So in order to make clock cult more "entertaining" ive done a few things which shall be explained below. 

#### Changelog

:cl:  
code/modules/antagonists/clockcult/clock_effects/servant_blocker.dm 
- Changed wording to cross to match
code/game/turfs/simulated/wall/misc_walls.dm
- Made Clock walls a bit more powerful
code/game/objects/items/stacks/sheets/sheet_types.dm
- Fixed some white space and added a new sheet stack for brass
code/game/machinery/doors/airlock_types.dm
- Made clock cult take less time some time and fixed some spelling errors on top of adding a new loot pool
 _maps/map_files/generic/City_of_Cogs.dmm 
- Added the loot pools to the map. They spawn random items round start to "help" clockies
-Added 3rd cog scarab to compensate for more space 
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
if you want to fix the issues with clock converting the entire crew you could also make conversion take increasing amounts of power the more cultists there are

just a thought

**Comment by MonsWTF** [comment]
To truly fix Cuckwork cultists, we need the lag at the start of the round to be fixed. To those "in on it" it's very obvious when a round is gonna be clockies because of that timeframe of 10-15 seconds where people can keep talking in OOC while the game is "Starting..."

**Comment by Readystorm546** [comment]
> To truly fix Cuckwork cultists, we need the lag at the start of the round to be fixed. To those "in on it" it's very obvious when a round is gonna be clockies because of that timeframe of 10-15 seconds where people can keep talking in OOC while the game is "Starting..."

Its pretty obvious what causes this. As clock cult does not load prior to the game starting. Instead clock cult loads when the game starts and it has to load so much shit. Just like the ash walkers. They dont spawn till after round start but you have to spawn in them so its different. #6727

File: code/game/turfs/simulated/wall/misc_walls.dm [hunk 1]
```diff
@@ -50,8 +50,8 @@
 	name = "clockwork wall"
 	desc = "A huge chunk of warm metal. The clanging of machinery emanates from within."
 	explosion_block = 2
-	hardness = 10
-	slicing_duration = 80
+	hardness = 15
```
Comment (SomeguyManperson) [inline code/game/turfs/simulated/wall/misc_walls.dm]: Lower hardness = stronger

Comment (Readystorm546) [reply code/game/turfs/simulated/wall/misc_walls.dm]: The more you know

File: code/game/machinery/doors/airlock_types.dm [hunk 2]
```diff
@@ -649,8 +649,7 @@
 	return 0
 
 /obj/machinery/door/airlock/clockwork/brass
-	glass = TRUE
-	opacity = 0
+	glass = FALSE
```
Comment (SomeguyManperson) [inline code/game/machinery/doors/airlock_types.dm]: Default clock airlock isn't glass but for some reason these are used on the map instead of those

Comment (Readystorm546) [reply code/game/machinery/doors/airlock_types.dm]: The more you know

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 3]
```diff
@@ -327,10 +327,10 @@ Credit where due:
...
 	<ul>\
 	<li><b>Ark Chamber:</b> Houses the Ark in the very center.</li>\
-	<li><b>Listening Station:</b> (Bottom Left Corner of circle) Contains intercoms, a telecomms relay, and a list of frequencies.</li>\
-	<li><b>Observation Room:</b> (Bottom Right Corner of circle) Contains six camera observers. These can be used to watch the station through its cameras, as well as to teleport down \
+	<li><b>Listening Station:</b> (Bottom Left Corner of Circle) Contains intercoms, a telecomms relay, and a list of frequencies.</li>\
+	<li><b>Observation Room:</b> (Bottom Right Corner of Circle) Contains six camera observers. These can be used to watch the station through its cameras, as well as to teleport down \
 	to most areas. To do this, use the Warp action while hovering over the tile you want to warp to.</li>\
-	<li><b>Infirmary:</b> (Uper Right Corner of circle) Contains sleepers and basic medical supplies for superficial wounds. The sleepers can consume Vitality to heal any occupants. \
+	<li><b>Infirmary:</b> (Uper Right Corner of Circle) Contains sleepers and basic medical supplies for superficial wounds. The sleepers can consume Vitality to heal any occupants. \
```
Comment (alexkar598) [inline code/game/gamemodes/clock_cult/clock_cult.dm:333]: Why Circle instead of circle?

Comment (Readystorm546) [reply code/game/gamemodes/clock_cult/clock_cult.dm:333]: As i recall the old map it was capitalized. But becides that i just want to catch the eye of the person who reads it.

**Pull request merged**